src/screens/FeedScreen/Content.js

```diff
@@ -34,11 +34,11 @@ const Content = ({
   item,
   onNewPollFetched,
   setHaveSeeMore,
-  hasComment
+  hasComment,
+  setIsShortText
 }) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Two new props `hasComment` and `setIsShortText` have been added to the component. Make sure these props are passed when using this component, otherwise it might lead to unexpected behavior.
```diff
+  hasComment,
+  setIsShortText
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->